bezier/src/quadratic_bezier.rs

```diff
@@ -246,6 +247,16 @@ impl QuadraticBezierSegment {
 
         return rect(min_x, min_y, max_x - min_x, max_y - min_y);
     }
+
+    /// Returns the smallest rectangle the curve is contained in
+    pub fn minimum_bounding_rect(&self) -> Rect {
+        let min_x = self.from.x.min(self.sample(self.find_x_minimum()).x).min(self.to.x);
```

Comment:
(reviewing from my phone so I may be missing something) I think that `find_x/y_minimum/maximum` ready takes care of comparing the inflection point(s) against self.from/to (or should). 

Better than my initial suggestion : ```let min_x = self.sample_x(self.find_x_minimum);```